src/styles/blocks/header.scss

```diff
@@ -0,0 +1,25 @@
+* {
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+}
```

Comment:
Move to normalize.css, and remove font-family, font family should be added to HTML tag